src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -104,14 +100,14 @@ const myAppConfig = {
 
 Amplify.configure(myAppConfig);
 ```
-[comment]: # (RENE, I realize this is likely outside the scope of this review, but the message in line 101 above within the code block (Before calling API,graphql...) isn't particularly clear. Is this something we can update to make clearer for the user?)
+
```

Comment:
Q from Jim during his initial edit pass - I realize this is likely outside the scope of this review, but the message in line 97 above within the code block - Before calling API,graphql... - isn't particularly clear. Is this something we can update to make clearer for the user?